urbanairship/push/payload.py

```diff
@@ -145,7 +150,7 @@ def ios(alert=None, badge=None, sound=None, content_available=False,
     if media_attachment is not None:
         payload['media_attachment'] = media_attachment
     if priority is not None:
-        if priority not in {10, 5}:
+        if priority not in set((10, 5)):
```

Comment:
Indeed, we are still officially supporting it with the library for now but we'll probably ditch it in an upcoming major release.